client/js/lounge.js

```diff
@@ -823,9 +823,21 @@ $(function() {
 	// Cycle through nicks for the current word, just like hitting "Tab"
 	$("#cycle-nicks").on("click", function() {
 		input.triggerHandler($.Event("keydown.tabcomplete", {which: 9}));
-		focus();
+		input.trigger("click").focus(); // Triggering click keeps the keyboard open on mobile
 	});
 
+	// Do not automatically focus on touch devices as this will open the virtual keyboard
+	var focus = $.noop;
```

Comment:
Wouldn't it be easier to do

var focus = function () {
  if (!("ontouchstart" in window... etc
}

rather than setting it to a noop?